.github/workflows/lib-injection.yml

```diff
@@ -83,8 +96,7 @@ jobs:
             -e PYTHONPATH=/lib-injection \
             -v $PWD/lib-injection:/lib-injection \
             ${{matrix.variant}}
-          # Package has to be built from source, wait a while
-          sleep 85
+          sleep 5
```

Comment:
Only 5 seconds, nice 😄. We're no longer building from source but we should still add a comment for the `sleep 5`.